### ISSUE TITLE
We no longer have the documentation generated by sphinx from the docstring

### ISSUE THREAD

**Issue by servoz** [populse/mia_processes#8]
We no longer have the documentation generated by sphinx from the docstring (example by clicking [on Module Index](https://populse.github.io/mia_processes/), for comparison see on [Module Index of populse_mia](https://populse.github.io/populse_mia/html/index.html)).
It may be that the change came after we [removed the built documents from the git sources](https://github.com/populse/mia_processes/issues/4).
I just noticed it, I'm opening this ticket to don't forget. I haven't taken the time to look yet...

**Comment by denisri** [comment]
We haven't upgraded the six module in the travis-ci config of this project. Thus this may be the cause, as I read in the doc build log:
```
WARNING: autodoc: failed to import module 'process_matlab' from module 'mia_processes'; the following exception was raised:
cannot import name 'collections_abc' from 'six.moves' (unknown location)
WARNING: autodoc: failed to import module 'process_mia' from module 'mia_processes'; the following exception was raised:
cannot import name 'collections_abc' from 'six.moves' (unknown location)
WARNING: autodoc: failed to import module 'spm' from module 'mia_processes.preprocess'; the following exception was raised:
cannot import name 'collections_abc' from 'six.moves' (unknown location)
WARNING: autodoc: failed to import module 'spm.spatial_preprocessing' from module 'mia_processes.preprocess'; the following exception was raised:
cannot import name 'collections_abc' from 'six.moves' (unknown location)
WARNING: autodoc: failed to import module 'tools' from module 'mia_processes'; the following exception was raised:
cannot import name 'collections_abc' from 'six.moves' (unknown location)
WARNING: autodoc: failed to import module 'tools.tools' from module 'mia_processes'; the following exception was raised:
cannot import name 'collections_abc' from 'six.moves' (unknown location)
```

**Comment by denisri** [comment]
OK that was the cause. It's fixed by f0f489975c6e9099d701a3084bac397ed5f1ab91.
